common/physics-system.cpp

```diff
@@ -203,7 +193,7 @@ eid PhysicsSystem::RayCastMousePick(
 	this->last_rayvalid = false;
 	this->last_entity_hit = 0;
 
-	CollisionBody* body = static_cast<CollisionBody*>(body_iter->second->getUserPointer());
+	auto* body = static_cast<CollisionBody*>(body_iter->second->getUserPointer());
```

Comment:
Suggestion from code quality tools to use auto since the name of the type is already on the initialization side.